sensors/triggers/triggers.go

```diff
@@ -26,9 +25,17 @@ import (
 
 func GetGroupVersionResource(obj *unstructured.Unstructured) schema.GroupVersionResource {
 	gvk := obj.GroupVersionKind()
+	lowercaseKind := strings.ToLower(gvk.Kind)
```

Comment:
Can we use this?

https://github.com/kubernetes/gengo/blob/master/namer/plural_namer.go#L54

Reply:
Yes, good point, changing now